app/src/main/java/com/bignerdranch/android/todoapp/ui/screens/LoginScreen/LoginPresenterimpl.java

```diff
@@ -24,10 +24,13 @@ public void dropView() {
     }
 
     @Override
-    public boolean loginMetod(DataHolder holder) {
-        if (holder.getDhUsername().equals(T_USERNAME) && holder.getDhPassword().equals(T_PASSWORD))
-            return true;
-        return false;
+    public boolean loginUserCheck(DataHolder holder) {
```

Comment:
ovdje stavljaš logiku koja od metoda ( gore što sam naveo da treba da napraviš) treba da se pozove